analysis/trigger_performance/01_readiness.md

```diff
@@ -0,0 +1,400 @@
+# Readiness trigger performance
+
+Reads in the readiness analysis by Joseph, does some BSRS,
```

Comment:
Who is Joseph? 
What does BSRS stand for?

Reply:
BSRS = bootstrap resample, aha. I will clarify.